xml/installation-installation-configure-deployer-repos.xml

```diff
@@ -0,0 +1,487 @@
+    directly copied to the &lcm; server, <quote>loop-mounted</quote> from an iso
+    image, or mounted from a remote server via NFS. Affected are &cloudos; and
+    &productname; &productnumber;. These are static repositories; they do not change or receive updates. See <xref linkend="sec.depl.adm_conf.repos.product"/> for setup
+    instructions.
+   </para>
+   <para>
+    <emphasis role="bold">Update and Pool Repositories</emphasis>: Update and
+    Pool repositories are provided by the &scc;. They contain all updates and
+    patches for the products and extensions. To make them available for
+    &cloud; they need to be mirrored from the &scc;. Since their content is
+    regularly updated, they must be kept in synchronization with &scc;. For
+    these purposes, &suse; provides either the &smtool; (&smt;) or the
+    &susemgr;.
+   </para>
+  </abstract>
+ </info>
+ <sect1 xml:id="sec.depl.adm_conf.repos.product">
+  <title>Copying the Product Media Repositories</title>
+  <para>
+   The files in the product repositories for &sls; and &cloud; do not
+   change, therefore they do not need to be synchronized with a remote
+   source. If you have installed the product media from a downloaded ISO image,
+   the product repositories will automatically be made available to the client
+   nodes and these steps can be skipped. Similarly, if you have configured the
+   &lcm; node as an SMT mirror, the product repositories will automatically be
+   made available to the client nodes and these steps can be skipped.
+   Otherwise, it is sufficient to either copy the data (from a remote host or
+   the installation media), to mount the product repository from a remote
+   server via <literal>NFS</literal>, or to loop mount a copy of the
+   installation images.
+   </para>
+   <para>
+    The following product media needs to reside in the specified
+    directories:
+   </para>
+   <table>
+    <title>Local Product Repositories for &cloud;</title>
+    <tgroup cols="2">
+     <colspec colnum="1" colname="1" colwidth="40*"/>
+     <colspec colnum="2" colname="2" colwidth="60*"/>
+     <thead>
+      <row>
+       <entry>
+        <para>
+         Repository
+        </para>
+       </entry>
+       <entry>
+        <para>
+         Directory
+        </para>
+       </entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry>
+        <para>
+         &productname; &productnumber; DVD #1
+        </para>
+       </entry>
+       <entry>
+        <para>
+         <filename>&www_dir;/repos/Cloud</filename>
+        </para>
+       </entry>
+      </row>
+     </tbody>
+    </tgroup>
+   </table>
+   <para>
+    The data can be copied by a variety of methods:
+   </para>
+   <variablelist>
+    <!-- sec.depl.adm_conf.repos.product.media -->
+    <varlistentry>
+     <term>Copying from the Installation Media</term>
+     <listitem>
+     <para>
+      We recommended using <command>rsync</command> for copying. If the
```

Comment:
do we still need that? why not mounting it?

Reply:
In the case a removable DVD it makes more sense to copy the media so that the DVD can be removed. Mounting an ISO or NFS serve is described later.